### ISSUE TITLE
[Bug] Improper Alignment of Blue Boxes in Mobile View in program-index.html

### ISSUE THREAD

**Issue by Raja-Abrar-Khan** [creativecommons/vocabulary#231]
## Description

In the mobile view, the blue content boxes ("Copyright" and "Better Internet") are not properly aligned and appear too large, disrupting the layout. This impacts readability and usability on smaller screens. This can be done by adjusting CSS . 

## Reproduction

1. Open the webpage in a mobile view (e.g., using developer tools or a mobile device).
2. Navigate to the section with the blue content boxes.
3. Observe the misalignment and large size of the boxes.
4. See error.

## Expectation

The blue boxes should be properly aligned and scaled down to fit mobile screens, maintaining readability and a clean layout.

## Screenshots
![Screenshot 2024-10-16 005112](https://github.com/user-attachments/assets/7153a431-beca-47ab-8286-8318daade03c)
![Screenshot 2024-10-16 005045](https://github.com/user-attachments/assets/2b831a23-a462-497c-9043-737366cbae52)

## Environment

 - Device: (_eg._ iPhone Xs; laptop)
 - OS: (_eg._ iOS 13.5; Fedora 32)
 - Browser: (_eg._ Safari; Firefox)
 - Version: (_eg._ 13; 73)
 - Other info: (_eg._ display resolution, ease-of-access settings)

## Additional context

There may be other layout issues on mobile, but  addressing this particular issue at a time is will be better to avoid confusion.

## Resolution

- [x] I would be interested in resolving this bug.

**Comment by Raja-Abrar-Khan** [comment]
@possumbilities  can you look into this issue this can b solved using some basic css .

**Comment by possumbilities** [comment]
@Raja-Abrar-Khan is the issue that they are not taking up equal vertical height?

**Comment by Raja-Abrar-Khan** [comment]
@possumbilities yes they are not taking the vertical height equally, as you can see in the height is looking all better in the normal screen but when it comes to the mobile screen it is not working as it was working in the main screen(desktop view) .

**Comment by kris70lesgo** [comment]
Hi @possumbilities @Raja-Abrar-Khan , I’ve reviewed the issue with the blue boxes in the mobile view. It seems that a PR (#236) was linked to resolve this. I will check if it has been merged and whether the issue is fully addressed. If not, I’d be happy to make the necessary CSS adjustments to ensure proper alignment and scaling on mobile views.

**Comment by possumbilities** [comment]
These boxes and this template are no longer utilized within the new vocabulary 2.0 setup. Closing.
